src/make-unbuffered-no-cache-request.js

```diff
@@ -29,8 +29,8 @@ export const makeUnbufferedNoCacheRequest = (
     // Build our main fetcher using the configured agent.
     return (
         superagent
-            .agent(agent)
             .get(url)
+            .agent(agent)
```

Comment:
Yup, this was not doing what we thought it should be.

Reply:
Wow, very interesting!

Reply:
When I was trying to get the flow errors and type definitions to reconcile, I noticed the difference. That led me to read the actual code and see that there is a difference. That helped me determine what the flow types should be and how to fix the errors.

What kind of API does this though? Where the ordering changes a method's meaning? Like, change the method name if that's the case. What the heck?